page_wrapper.go

```diff
@@ -103,6 +103,10 @@ func (p *pageWrapper) Sleep(time float64) {
 	p.Page.WaitForTimeout(time)
 }
 
+func (p *pageWrapper) WaitForTimeout(time float64) {
```

Comment:
Should we remove the Sleep?

Reply:
I am not sure, I just left it as we are extending their functionality and I thought keeping already written APIs isn't harmful. But I totally understand how we have two functions with same functionality, so if you think we should remove 'Sleep', I'll remove it.